rotkehlchen/chain/substrate/utils.py

```diff
@@ -42,3 +49,49 @@ def is_valid_substrate_address(
     except ValueError:
         return False
     return True
+
+
+def get_kusama_address_from_public_key(public_key: SubstratePublicKey) -> KusamaAddress:
+    """Return a valid Kusama address given a Substrate public key.
+
+    Public key: 32 len str, leading '0x' not needed.
+
+    May raise:
+    - AttributeError: if public key is not a string.
+    - TypeError: if ss58_format is not an int.
+    - ValueError: if public key is not 32 bytes long or the ss58_format is not
+    a valid int.
+    """
+    return get_substrate_address_from_public_key(
+        chain=SubstrateChain.KUSAMA,
+        public_key=public_key,
+    )
+
+
+def get_substrate_address_from_public_key(
+        chain: SubstrateChain,
+        public_key: SubstratePublicKey,
+) -> SubstrateAddress:
+    """Return a valid address for the given Substrate chain and public key.
+
+    Public key: 32 len str, leading '0x' not needed.
+
+    Utils:
```

Comment:
what do you mean by utils here? That this is where you got the code logic from? If so then just write that. If it's not useful then just don't add it in the docstring.

Reply:
It is a very web useful for validating Substrate-related data conversions. I have it bookmarked so I will remove it.